apps/src/applab/applab.js

```diff
@@ -161,6 +162,11 @@ function loadLevel() {
   for (var key in level.scale) {
     Applab.scale[key] = level.scale[key];
   }
+
+  if (Applab.makerlabEnabled) {
+    var BoardController = require('../makerlab/BoardController');
```

Comment:
Hmmm....that does seem unfortunate. The reason I like to have requires at the top is it makes it easier to understand a module's dependencies. In this case we have a dependency that's conditional on runtime config.

One option would be to just make BoardController resilient to the case where ChromeSerialPort is null, but that doesn't seem ideal.

I'm not sure yet what the best way to solve this is :(